geo/src/algorithm/euclidean_length.rs

```diff
@@ -35,18 +36,18 @@ where
 
 impl<T> EuclideanLength<T> for LineString<T>
 where
-    T: Float,
+    T: Float + Sum,
 {
     fn euclidean_length(&self) -> T {
         self.lines()
             .map(|line| line.euclidean_length())
-            .fold(T::zero(), |total_length, length| total_length + length)
+            .sum()
     }
 }
 
 impl<T> EuclideanLength<T> for MultiLineString<T>
 where
-    T: Float,
```

Comment:
Did you mean to convert the `fold` usage below into `sum`?

Reply:
It's needed because of the `Sum` bound on `EuclideanLength` for `LineString`.